src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstaticimport/InputAvoidStaticImportDefault.java

```diff
@@ -22,7 +22,6 @@
 import javax.swing.BorderFactory;
 import static java.io.File.listRoots;
 
-import static javax.swing.WindowConstants.*;
```

Comment:
Why is this removed?

Reply:
@strkkk We are not expecting inputs to be changed in these issues.

Reply:
@rnveach it is a duplicated import.
Line 26 in same file contains exactly same import, so this line is redundant.
It is out of scope of this issue, but I asked @gaurabdg to remove it,